src/Symfony/Component/CssSelector/Parser/Parser.php

```diff
@@ -128,16 +121,7 @@ private function parseSelectorList(TokenStream $stream)
         return $selectors;
     }
 
-    /**
-     * Parses next selector or combined node.
```

Comment:
This comment might be useful? (I don't know the class so might be wrong)

The same applies to the other methods of that class, but if it's expected that all methods parse the "*next […]*" then feel free to ignore my comment.